packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -410,10 +427,14 @@ describe('lion-select-rich', () => {
           </lion-options>
         </lion-select-rich>
       `);
+      // The default is min, so we override that behavior here
+      el._overlayCtrl.inheritsReferenceWidth = 'full';
```

Comment:
We should probably always use `updateConfig` as a best practice. 
I think those props that are directly available on the controller are mainly there for backwards compatibiliy and if we want to keep them, they should be readonly.

Setting props on overlayCtrl directly is not guaranteed to properly reload init/teardown phases etc.

Reply:
Yeah but it's a test so it doesn't matter right? All I wanna test here is if the hasNoDefault case overrides the inherits width but I can't properly do this if the initial inherits width is the same width as the override

Reply:
Well, of course we should use the right public apis preferrably when possible, since tests also serve as examples.
So far we don't do it (change props directly on the controller) in our demos/tests at least, but I agree this should be better documented: we should explicitly say that changing a config like this doesn't trigger the right init and teardown actions.

Don't want to nitpick too much, so if you want to do this later, that's fine 👍